patches/components-crash-core-app-breakpad_linux.cc.patch

```diff
@@ -5,13 +5,13 @@ index 81578d1e0928a8f67922054864ec777cf2d0ddfe..94997072e4399a27e83fad6258cc3e5d
 @@ -1461,7 +1461,7 @@ bool IsValidCrashReportId(const char* buf, size_t bytes_read,
    return my_strcmp(buf, "_sys_cr_finished") == 0;
  #else
-   for (size_t i = 0; i < bytes_read; ++i) {
--    if (!my_isxdigit(buf[i]))
-+    if (!my_isxdigit(buf[i]) && buf[i] != '-')
-       return false;
-   }
-   return true;
-@@ -1954,7 +1954,7 @@ void HandleCrashDump(const BreakpadInfo& info) {
+   return base::ranges::all_of(base::span(buf, bytes_read),
+-                              base::IsHexDigit<char>);
++                              IsHexDigitOrDash<char>);
```

Comment:
where does `IsHexDigitOrDash` come from?

Reply:
in any case can we chromium_src override `IsHexDigit` to call `IsHexDigitOrDash`?

Reply:
`IsHexDigitOrDash` was defined in `chromium_src` override, but the file got dropped, which highlight an issue with this whole patch: the code isn't even being built. I would be in favour of deleting this whole patch if everyone agreed.

Tagging @goodov here, because I had brought up this change with him. As he had pointed out in slack, these files are not being built due a change introduced just over a year ago https://github.com/chromium/chromium/commit/7c44905e4a83c8c4811867c5bb27d10d8a086b34.

Reply:
The file is not used in the build. https://source.chromium.org/chromium/chromium/src/+/main:components/crash/core/app/BUILD.gn;l=85

just remove the patch.

Reply:
fixed in 7cc9f745843e99203907c73fd70598ea6ed64264.